frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/ClickBehaviorSidebarMainView/ClickBehaviorSidebarMainView.tsx

```diff
@@ -6,6 +6,8 @@ import type {
   DashboardOrderedCard,
   ClickBehavior,
 } from "metabase-types/api";
+import { isActionDashboardCard } from "metabase-types/guards";
```

Comment:
This seems to be the only place that imports from `metabase-types/guards` rather than `metabase-types/guards/some-files`. I think we should have only a single type of import to make more consistent and avoid confusion.